stdlib/public/core/SequenceWrapper.swift

```diff
@@ -110,10 +106,6 @@ extension _SequenceWrapper
     return try _base.prefix(while: predicate)
   }
   
-  public func suffix(_ maxLength: Int) -> SubSequence {
```

Comment:
Is it no longer necessary?

Reply:
It's redundant, because I merged two extensions into a single extension. The extension of `_SequenceWrapper`

    where SubSequence == Base.SubSequence, Element == Base.Element

is just an extension of `_SequenceWrapper` now.